rcl/CMakeLists.txt

```diff
@@ -3,14 +3,68 @@ cmake_minimum_required(VERSION 2.8.3)
 project(rcl)
 
 find_package(ament_cmake REQUIRED)
+find_package(rcl_interfaces REQUIRED)
+find_package(rmw REQUIRED)
+find_package(rmw_implementation REQUIRED)
+find_package(rmw_implementation_cmake REQUIRED)
+find_package(rosidl_generator_c REQUIRED)
 
+include_directories(include)
+
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
```

Comment:
I had trouble with the line `#include <time.h>` when I enabled `-std=c11`, but I never tracked it down to a bug report or tried with a newer version of gcc.

Windows doesn't support C11 and I can't find any indication that they plan to improve on that: http://herbsutter.com/2012/05/03/reader-qa-what-about-vc-and-c99/

I can tell you that they would need some extra features or compiler support to implement the C11 atomic api as described using their atomics primitives.

I haven't run into any C11 feature that I need yet and I had strange compiler behavior on Linux with gcc when using `-std=c11` (but it works fine on OS X), so I don't pass the flag.